client/src/components/product_overview/Carousel.jsx

```diff
@@ -1,40 +1,54 @@
 /* eslint-disable react/prop-types */
 import React from 'react';
-import { MdArrowForward, MdArrowBack, MdFullscreen, MdArrowDropDown } from 'react-icons/md';
+import { MdArrowForward, MdArrowBack, MdArrowDropDown } from 'react-icons/md';
+import List from '@material-ui/core/List';
+import ListItem from '@material-ui/core/ListItem';
 
 class Carousel extends React.Component {
   constructor (props) {
     super(props);
     this.state = {
+      expandButtonClicked: false
     };
   }
 
+  handleFullScreen () {
+    this.setState({ expandButtonClicked: true });
+  }
+
   render () {
+    if (this.props.currentStyle.photos) {
+      console.log('♟', this.props.currentStyle.photos.slice(0, 7));
+    }
     return (
       <div className="carousel">
 
-        <div className="carousel_container" style={{ backgroundImage: `url(${this.props.currentImage})` }} >
+        <div className={`carousel_container ${this.state.expandButtonClicked ? 'expand_clicked' : ''}`} style={{ backgroundImage: `url(${this.props.currentImage})` }} >
 
-          <div className="tumbnails_container">
+          <List className="thumbnails_container" style={{ maxHeight: 450, overflow: 'auto' }}>
```

Comment:
Consider not using inline styles to avoid confusion since you also are using a styles sheet